terraform/bootstrap/create_service_account.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
```

Comment:
thought (non-blocking): I feel like we might want to move the bootstrap out to a `devtools-iac` or other repo and leave just a pure Terraform module here.

Marking non-blocking. It is great to get this in place somewhere, and we can figure out if we want to move it later.

Reply:
Created issue to refactor bootstrap based on all of these comments.